aconcli/cmd/restart.go

```diff
@@ -61,4 +64,6 @@ func init() {
 
 	restartCmd.Flags().Uint64VarP(&timeout, "timeout", "t", 30,
 		"optional timeout in seconds to wait before restarting the container")
+	restartCmd.Flags().BoolVar(&nologin, "nologin", false,
+		"if set, login as an anonymous user")
```

Comment:
This `nologin` seems set by multiple commands. Why not use a global (persistent?) flag?

Reply:
resolved as a persistant flag